src/desktop/apps/article/routes.tsx

```diff
@@ -157,6 +158,7 @@ export const index = async (req, res, next) => {
     const { CURRENT_USER, IS_MOBILE, IS_TABLET } = res.locals.sd
     const isMobile = IS_MOBILE
     const isTablet = IS_TABLET
+    const { isEigen } = buildServerAppContext(req, res)
```

Comment:
Note: Using `buildServerAppContext` here to check if we're on Eigen (specific user agent is set)